docs/references/api.md

```diff
@@ -290,44 +335,36 @@ This will be called to get the answer to users' questions.
     "retmsg": "success"
 }
 ```
-
-- **answer**: The replay of the chat bot.
-- **reference**: 
-    - chunks: Every item in it refer to the corresponding message in answer. 
-        - content_with_weight: The content of chunk.
-        - docnm_kwd: the document name.
-        - img_id: the image id of the chunk. It is an optional field only for PDF/pptx/picture. And accessed by 'GET' /document/get/\<id\>.
-        - positions: [page_number, [upleft corner(x, y)], [right bottom(x, y)]], the chunk position, only for PDF.
-        - similarity: the hybrid similarity.
-        - term_similarity: keyword simimlarity
-        - vector_similarity: embedding similarity
-    - doc_aggs:
-        - doc_id: the document can be accessed by 'GET' /document/get/\<id\>
-        - doc_name: the file name
-        - count: the chunk number hit in this document.
-    
+</details>    
 ## Get document content or image
 
-This is usually used when display content of citation.
-### Path: /api/document/get/\<id\>
-### Method: GET
+- Path: `/api/document/get/<id>`
+- Method: GET
+
+This method retrieves the content or a specific image in a document. Used if you intend to display the content of a citation.
+
 
 ## Upload file
 
-This is usually used when upload a file to.
-### Path: /api/document/upload/
-### Method: POST
+- Path: `/api/document/upload/`
+- Method: POST
+
+This method uploads a specific file to a specified knowledge base.
 
-### Parameter:
 
-| name      | type   | optional | description                                             |
-|-----------|--------|----------|---------------------------------------------------------|
-| file      | file   | No       | Upload file.                                            |
-| kb_name   | string | No       | Choose the upload knowledge base name.                  |
-| parser_id | string | Yes      | Choose the parsing method.                              |
-| run       | string | Yes      | Parsing will start automatically when the value is "1". |
+### Parameter
+
+|   Name      | Type   | Required | Description                                             |
+|-------------|--------|----------|---------------------------------------------------------|
+| `file`      | file   | Yes      | The file to upload.                                     |
+| `kb_name`   | string | Yes      | The name of the knowledge base to upload the file to.   |
+| `parser_id` | string |  No      | The parsing method (chunk template) to use.             |
+| `run`       | string |  No      | - 1: Automatically start file parsing.                  |
```

Comment:
What if I set `run` to -1 without setting `parser_id`?

Reply:
naive:General,
qa:Q&A
manual:Manual
table:Table
paper:Paper
book:Book
laws:Laws
presentation:Presentation
picture:Picture
one:One